docs/modules/ROOT/pages/how-to-guides/proc_release_application.adoc

```diff
@@ -39,11 +37,12 @@ NOTE: The `ReleasePlan` and `ReleasePlanAdmission` is an agreement between the d
 
 .*Prerequisites*
 * You have an existing Development and Managed workspace.
-* Ensure you have installed oc.
+* Ensure you have installed `oc`.
+* You have completed the steps listed in the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/[Getting started in the CLI] page.
 
 .*Procedures*
 
-. In the development workspace, create a release_plan.yaml object by running:
+. Create a `release_plan.yaml` object locally.
```

Comment:
The command below should probably be removed from here - the command is for applying the file once it exists and also, it's shown again later on, around line 80.

`oc apply -f releaseplan.yaml -n dev`